wiki/debugging.md

```diff
@@ -0,0 +1,76 @@
+# Debugging data issues
+
+Many issues can arise in the Metro galaxy, from the shallowest part of the frontend to the deepest end of the backend. Tackling data inaccuracies requires a strategic approach. This guide traces an easeful, step-by-step process for efficient debugging. 
+
+DON'T PANIC. 
+
+## Identifying the point of failure
+
+[point to the charts in the overview that describe the data pipeline]
+
+Data travels along a complicated path to arrive in Metro Councilmatic. A disruption in the data pipeline can happen at four different points. 
+
+1. Legistar - the Legistar API or web interface did not provide accurate data 
+2. OpenCivicData API - the DataMade scrapers did not properly port data to the API 
```

Comment:
is it worth mentioning that this is also a database, underneath it all?

Reply:
should we intro pupa + python-legistar here at all?

Reply:
Let's mention both these things at the bottom of "Step 4" (below). These opening four points should not overwhelm the reader with too much detail, I think.